src/audio/src/src.c

```diff
@@ -602,28 +627,22 @@ static int src_set_params(struct processing_module *mod)
 	return 0;
 }
 
-static void src_set_sink_params(struct comp_dev *dev, struct comp_buffer __sparse_cache *sinkb) {}
+static void src_set_sink_params(struct comp_dev *dev, struct sink __sparse_cache *sink,
+				struct comp_buffer __sparse_cache *sinkb)
+{
+	/* empty */
```

Comment:
duplicate of above signature

Reply:
Can you explain?

Reply:
Ah nvm, I missed "Sink" somehow in the name of above function